src/cfg.h

```diff
@@ -71,6 +74,26 @@ class AssignmentNode : public Node<LatticeType> {
     }
 };
 
+template<typename LatticeType>
+class IfNode : public Node<LatticeType>{
+public:
+    std::shared_ptr<Expression> expression;
+
+    IfNode(std::shared_ptr<Expression> expression) : expression(expression){}
+
+    void accept(CfgVisitor<LatticeType>& visitor){
+        visitor.visit_if(*this);
+    }
+
+    void dotPrint(std::ostream &os){
+        os << (unsigned long long int)this << "[label = \" " << expression->dotPrint() << "\"]\n";
+        for (auto& succ : this->successors){
+            os <<(unsigned long long int)this << "->" << (unsigned long long int)succ.get() << "\n";
+            succ->dotPrint(os);
+        }
+    }
```

Comment:
This code should not be here. Just remove it

Reply:
👍